pkgs/top-level/all-packages.nix

```diff
@@ -9212,7 +9202,7 @@ with pkgs;
   goocanvas3 = callPackage ../development/libraries/goocanvas/3.x.nix { };
   grpc = darwin.apple_sdk_11_0.callPackage ../development/libraries/grpc {
     stdenv = if stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isx86_64
-      then overrideSDK darwin.apple_sdk_11_0.stdenv { darwinMinVersion = "10.13"; }
+      then overrideSDK stdenv { darwinMinVersion = "10.13"; }
       else stdenv;
```

Comment:
Not necessary.

Reply:
I guess dropping `overrideSDK` here will result in 500+ rebuilds...

Reply:
Maybe best to do it in a separate PR and handle the `darwin.apple_sdk_11_0.callPackage` for it there too.

Reply:
Moved to #370670 and a subsequent PR.